NeoML/Python/neoml/Dnn/ObjectNormalization.py

```diff
@@ -48,38 +63,38 @@ def __init__(self, input_layer, epsilon=0.00001, name=None):
 
     @property
     def epsilon(self):
-        """
+        """Gets the small value used to avoid division by zero.
         """
         return self._internal.get_epsilon()
 
     @epsilon.setter
     def epsilon(self, epsilon):
-        """
+        """Sets the small value used to avoid division by zero.
         """
         if float(epsilon) <= 0:
             raise ValueError('The `epsilon` must be > 0.')
         self._internal.set_epsilon(float(epsilon))
 
     @property
     def scale(self):
-        """
+        """Gets scale, one of the trainable parameters in the formula.
```

Comment:
It's important to mention here that `scale` and `bias` are trainable parameters (blobs) and their sizes

Reply:
Added blob size description.